static/app/components/progressRing.tsx

```diff
@@ -98,7 +99,7 @@ function ProgressRing({
 
   let textNode = (
     <TextComponent
-      key={text?.toString()}
+      key={Object(text) !== text ? text?.toString() : uniqueId()}
```

Comment:
it is not always that a text is of type primitive. this change avoids `[object, object]` as a key here